### DISCUSSION TITLE
[Scam Alert] Unique Finance یونیک فایننس

### DISCUSSION THREAD

**Discussion by Webamoozcom** [Webamoozcom/warning-list#27] [Announcements]
<div dir="rtl">

**Project Name:** 
Unique Finance

**نام پروژه:** 
یونیک فایننس 

**نماد پروژه:**
UNQT

**اتفاقی که افتاده است:**
نگاهی به یونیک فاینانس و گفته های بنیان گذاران
احتمالا شما که در حال مطالعه این مقاله هستید درباره این پروژه زیاد شنیده اید. بنیانگذاران این پروژه ادعا کرده اند که ” ما سود زیادی از بورس کسب می کنیم و می خواهیم نیمی از این سود را به کاربرانمان اهدا کنیم. “. بنیانگذاران این مجموعه ادعا کرده اند سودی که می دهند دائمی می باشد. طبق دیگر اطلاعاتی که مطالعه شد تقریبا این سود 4 برابر سود بانک ها در ایران می باشد. داشتن زیر مجموعه در یونیک فاینانس به بیشتر سوددهی کمک می کند. بنیانگذاران یونیک فایننس ادعا کرده اند که با داشتن زیر مجموعه می توانید تا ماهی 100 هزار دلار نیز درآمد داشته باشید. شاید با شنیدن این ادعا کمی تعجب کنید اما باید بدانید که تمامی این حرف ها تنها برای جذب کاربر زده می شود و هیچ اعتبار دیگری ندارد.

در ادامه قصد داریم تا سری به سایت یونیک فاینانس به آدرس Http://Unique.Finance/ بزنیم و کمی سایت آن را تحلیل کنیم. اگر قصد دارید تا درباره این مقاله اطلاعات بیشتری کسب کنید با ما همراه باشید.

نیم نگاهی به سایت یونیک فاینانس

برای اینکه کمی بخواهید بیشتر با این پروژه یونیک فاینانس آشنا شوید می توانید به سایت رسمی آن مراجعه کنید. ما نیز در این قسمت از مقاله این کار را کرده ایم و به سراغ سایت این پروژه رفته ایم. در اولین قدم به بخش ” درباره ما ” یا ” About Us ” این سایت مراجعه می کنیم. با مراجعه به این قسمت با اطلاعات مختلفی رو به رو می شویم. یکی از این اطلاعات نام بنیانگذاران این پروژه می باشد. نام تمامی آن ها غیر آشنا می باشد. برای اینکه بهتر با آن ها آشنا شویم، نام تک به تک آن ها را در گوگل جستجو می کنیم. با هر عبارتی که نام اولین بنیانگذار یونیک فاینانس را جستجو کردیم، برای ما اطلاعاتی از آن فرد بارگذاری نکرد. اگر یک پروژه واقعی باشد و به کاربران خود سودخوبی بدهد، قطعا سروصدا های زیادی می کند و و قطعا توسعه دهندگان آن سمتی خواهند داشت. تنها با تکیه بر این موارد نمی توان گفت که این پروژه کلاهبرداری است و قطعا باید برای اثبات این موضوع به موارد دیگری نیز نگاه بیندازیم.

تا به حال به این فکر کرده اید که این سایت چرا هیچگونه شماره تماسی از خود به جای نگذاشته است؟ آیا تا به حال اگر به سایت یونیک فاینانس مراجعه کرده باشید، به این موضوع دقت کرده اید؟

زمانی که از توسعه دهندگان یونیک فاینانس در مکاتبه های صورت گرفته درخواست شماره تماس شد، آنها در پاسخ گفتند که: به دلیل تماس مکرر افراد ایرانی، نمی توانیم شماره تلفن را اختیار کسی قرار دهیم. سوالی که ما از شما داریم این است که آیا امکان اینکه یک پروژه بخواهد اینچنین حرفی را بزند وجود دارد یا خیر؟

در قسمت بعدی قصد داریم تا نگاهی به سخنان کاربرانی بیندازیم که درباره این پروژه نظر داده اند و تقریبا تجربه دار هستند. در ادامه همراه ما باشید.

برای شناخت بهتر شرکت یونیک فایننس، باید به وبسایت این شرکت سر بزنیم. تعریفی که در صفحه اول وبسایت یونیک فایننس نوشته شده، به صورت زیر است:
با دریافت و مشاهده دوره آموزش ارزهای دیجیتال، به دنیای جذاب و پرسود رمز ارزها وارد شوید.
“یونیک فایننس دارای گروهی از معامله‌گران با مهارت بالا است و این شرکت، یک شرکت معمولی در حوزه سرمایه‌گذاری نیست. یونیک فایننس بیش از 10 سال است که در بازارهای مالی دارای تجربه است و این تجربه‌ها باعث شده که در مقابل رقیبان خود متنوع‌تر و بهتر عمل کنیم.
شعار شرکت ما «ما می‌توانیم» است. با تلاش فراوان در راستای همین شعار، توانسته‌ایم تجربه‌های شگفت‌انگیزی به دست آوریم.
باید بدانید که موفقیت فقط به معنی کسب سود در زمان کوتاه نیست؛ در واقع به معنی به دست آوردن اعتباری از مشتریان بلند‌مدت می‌باشد که از اعتمادی که کرده‌اند و از تصمیم‌های مالی ما برای آنها رضایت دارند. تیم ما با پشتکار زیاد تلاش می‌کند که در آینده موفق‌تر باشد؛ زیرا هر روز که می‌گذرد ما تجربه تازه‌تری به دست می‌آوریم. یک تصمیم مالی بر اساس یک حقیقت و تجزیه و تحلیل وقایع گذشته، آینده و حال گرفته می‌شود.
ما با کمک متخصصان با تجربه و ماهر، بازار را مورد ارزیابی قرار می‌دهیم.”

به چند دلیل می‌توان گفت یونیک فایننس، کلاهبرداری است:
•	این شرکت از هیچ کشوری مجوز فعالیت ندارد.
•	این شرکت ادعا دارد که کارگزاری می‌باشد، اما هیچ مدرک و تأییدیه‌ای برای این اظهارنظر وجود ندارد.
•	با سرچ کردن نام افراد توسعه‌دهندۀ این شرکت در گوگل، به هیچ نتیجه‌ای دست پیدا نخواهید کرد.
•	هیچ سندی مبنی بر خرید سهام توسط یونیک فایننس وجود ندارد.
•	خرید و فروش در لحظه، عملاً غیرقابل انجام است؛ زیرا با پرداخت کارمزد، کاربر به جای سود، ضرر می‌کند. بنابراین ادعای داشتن یک ربات هوشمند برای خرید و فروش لحظه‌ای رد می‌شود.
•	هیچ شماره تماس و آدرس شرکت مرکزی از یونیک فاینانس وجود ندارد.

بررسی تاریخچه یونیک فایننس

یونیک فایننس خود را یک شرکت سرمایه‌گذاری یا همان شرکت مدیریت مالی معرفی و محل کار خود را در زوریخ سوئیس بیان کرده است. این شرکت مدعی است که دارای یک گروه از معامله‌کنندگان باتجربه و ماهر است که بیش از ده سال در بازار های مالی در حال کار کردن هستند. شرکت یونیک فایننس هنگامی شروع به کار کرد که یک رسانه تجارت بین سرمایه‌گذاران و بازار سهام شمرده می‌شد. با گذشتن از قرن بیستم، یونیک فایننس با استفاده از اینترنت، تبدیل به یکی از شرکت‌های تجاری الکترونیک شد. در قرن بیست و یکم نیز این شرکت به یکی از پرطرفدارترین شرکت‌ها برای سرمایه‌گذاری و مدیریت سرمایه در بازار جهانی تبدیل شده است. یونیک فایننس در سال 2006 تیم تجاری آمریکای شمالی را به وجود آورد و با موفقیت توانست یک مشاوره آنلاین را در شمال آمریکا و کانادا که روی بورس اوراق بهادار نیویورک تمرکز دارد، افتتاح کند.
طبق ادعای این شرکت، یونیک فایننس در بازارهای مختلف مالی جهانی فعالیت دارد که بعضی از آنها عبارت‌اند از:
•	بورس اوراق بهادار نیویورک؛ شامل بورس نیویورک و بورس نزدک (NASDAQ)
•	بورس اوراق بهادار لندن
•	بورس اوراق بهادار فرانکفورت
•	بورس اوراق بهادار لوگزامبورگ
•	بورس اوراق بهادار تورنتو
•	بورس اوراق بهادار استرالیا
توجه: تمامی توضیحات بالا بر اساس ادعای شرکت یونیک فایننس است.
نحوه عملکرد یونیک فایننس
 
سر شاخه‌های اصلی شرکت یونیک فایننس در اکثر اوقات به صورت یک فرد تحصیل کرده در مکان‌های شیک هستند. این افراد با حفظ آراستگی هرچه ‌تمام‌تر، تلاش می‌کنند که اعتماد کاربران را به دست آورند. با توجه به گفتۀ خود شرکت یونیک فایننس، این شرکت کارگزار بورس است و توسط ربات به خرید و فروش می‌پردازد. همچنین این شرکت ادعا دارد که سهام‌های شرکت اپل و بی‌ام‌و و غیره را خرید و فروش می‌کند. در توضیحات این شرکت آمده است که با کسب زیرمجموعه می‌توان به سود بیشتری دست پیدا کرد؛ اما کسب زیرمجموعه الزامی نیست و با نداشتن زیرمجموعه نیز هر فرد روزانه حدود 3 دلار سود می‌کند که مقداری از این سود به خود شرکت تعلق خواهد گرفت. هر کاربر بعد از عضویت در شرکت می‌تواند سه نفر را به صورت زیرمجموعه مستقیم داشته باشد و به ازای 5000 دلار سرمایه‌گذاری هر سه زیرشاخه، 1000 دلار به سر شاخه (دعوت‌کننده کاربر) خواهد رسید. هر کاربر برای شروع کار خود در شرکت یونیک فایننس باید 1000 دلار سرمایه‌گذاری (خرید) انجام دهد تا به صورت ماهیانه به سود دست پیدا کند.
جالب است بدانید که کاربر تا 2 سال نمی‌تواند به اصل پول خودش دسترسی داشته باشد!
ادعاهای مخالفان یونیک فایننس
 
مخالفان طرح و شرکت یونیک فایننس اعتقاد دارند که این سایت همانند سایت‌های هایپ (HYIP) و سیستم آن، پانزی (Ponzi) است.
در واقع یعنی سودی که به شما داده می‌شود، از طریق ورود پول کاربران جدیدتر است. این سایت‌ها برای به دست آوردن اعتماد افراد، چند ماه سود سرمایه‌گذاری را پرداخت می‌کنند.
برای درک بهتر کلاهبرداری این شرکت، بهتر است به ادعای خود شرکت رجوع کنید. این شرکت می‌گوید در بازارهای مالی به سرمایه‌گذاری و خرید و فروش می‌پردازد، اما زمانی که به جست‌وجوی بیشتری بپردازید، در خواهید یافت که هیچ مدرکی مبنی بر معاملات این شرکت وجود ندارد. داشتن گواهی SSL و همچنین داشتن نماد از برخی سایت‌های امنیتی همانند (مکافی، نورتون و کسپراسکای) دلیلی بر معتبر بودن سایت نیست؛ زیرا هر فرد می‌تواند با پرداخت کردن مبلغ کمی، به این گواهی‌ها دست پیدا کند. طبق قوانین تجارت، هر شرکتی که بخواهد در هر کشوری به فعالیت تجاری بپردازد، باید دارای مجوز رسمی از نهادهای مالی مربوطۀ آن کشور باشد.

اخطار FINMA برای سرمایه‌گذاری در یونیک فایننس
 
FINMA یک نهاد حکومتی در کشور سوئیس است که بر بانکداری و بازار مالی این کشور نظارت‌ می‌کند. یونیک فایننس در لیست اخطارهای این نهاد نظارتی قرار دارد. برخی از افراد معتقدند که یونیک فایننس به دلیل مسائل مالیاتی در این لیست قرار گرفته است، اما فینما گفته است که یونیک فایننس به دلیل کلاهبرداری و تظاهر به سوئیسی بودن، در این لیست قرار دارد.
قرار گرفتن یونیک فایننس در لیست فینما، می‌تواند دلیل قانع‌کننده‌ای برای کلاهبرداری بودن آن باشد.
جالب است بدانید که در کشور ما نیز چند نفر از سرشاخه‌های این شرکت دستگیر شده‌اند. در سال گذشته 6 سرشاخه در یزد و خراسان رضوی دستگیر شدند. این 6 سرشاخه توانسته بودند 2500 عضو برای خود جذب کنند.
به گزارش روابط عمومی دادگستری، این افراد با جذب سرمایه دیگران و ترغیب کردن آنها به شبکه‌سازی، توانسته بودند سرمایه‌ای بالغ بر 155 میلیون دلار را جذب شبکۀ خود کنند و مقدار قابل‌توجهی از آن را، از کشور خارج کنند.

سخن پایانی

روزانه طرح‌های کلاهبرداری بسیاری برای جذب سرمایه مردم به وجود می‌آید. مهم این است که همیشه به یاد داشته باشید که برای شروع یک سرمایه‌گذاری، تمام جوانب آن را مورد بررسی قرار دهید. این نکته را هم فراموش نکنید که هیچ سودی را نمی‌توانید به راحتی بدست آورید. تصمیم شما برای سرمایه‌گذاری کردن یا نکردن در شرکت یونیک فایننس، به عهدۀ خود شما است. اما با توضیحات داده شده، مشخص است که فعالیت این شرکت کاملا کلاهبرداری و پانزی است.

**قوانینی که نقض شده است:**

**نوع هشدار:**
هشدار کلاهبرداری

**نوع کلاهبرداری:** 
طرح پانزی، طرح هرمی

**وضعیت کلاهبرداری:**
در جریان

**نام کلاهبرداران:**

**نام شرکت(ها):**

**آدرس وب سایت(ها):**
http://unique.finance/

**برآورد مبلغ کلاهبرداری:**

**منابع:**
https://www.zoomarz.com/uniqe-finance

Complaints against pyramid scam Unique Finance 
در انتشار اسناد کانال و آگاه سازی عموم مردم ما را یاری کنید

لینک گروه شاکیان یونیک فاینانس👇
https://t.me/shakianuniquefinance

**جمع بندی:**

**Comment by Webamoozcom** [comment]
Confirmation: Account Closed
Hello
This is to confirm that based on your request, your account with Unique Finance has been canceled.
Any due refund will be processed according to our terms & conditions.
You must now download this guide note and follow the instructions in it to get your refund. You can follow the progress from within your account.
Thank you,
Unique Finance Legal Team

[4_5847009653178761960.pdf](https://github.com/Webamoozcom/warning-list/files/7145448/4_5847009653178761960.pdf)


https://t.me/warning_list/87

**Comment by Webamoozcom** [comment]
سلام به همگی بعضی از دوستان بعد از دوسال که قراردادشون تو شرکت کلاهبردار یونیک فایننس تموم شد برای گرفتن پولشون این پی دی اف رو براشون ارسال کردن خوشحال میشم آقای جرجندی یه بررسی بکن واگه دوست داشتن راجع بهش پست بزارن چون اصل کلاهبرداری هستش و قوانین من در آوردی خیلی توشه یعنی یه هفت خان رستم هستش که رستم هم باشی نمیتونی ردش کن و مصداق بارز کلاهبرداریه

**Comment by Webamoozcom** [comment]
دوست عزیز تو چند مرحله چند مدل مدارک خواستن که ارسال بشه همه هم باید ترجمه رسمی به انگلیسی بشه و با پست dhl ارسال بشه و جالب اینه که بعد از ارسالش گفتن ایمیل نزنین برای جوابش و صبر کنید چون دیگه از اینجا به بعد به شرکت مربوط نیست یعنی برای دریافت هزار دلار که هیچ ضمانتی براش وجود نداره تازه باید کلی هم خرج کنی و جالبه واقعا

**Comment by Webamoozcom** [comment]
Complaints against pyramid scam Unique Finance 
در انتشار اسناد کانال و آگاه سازی عموم مردم ما را یاری کنید

لینک گروه شاکیان یونیک فاینانس👇

https://t.me/shakianuniquefinance

**Comment by Webamoozcom** [comment]
کانال استاد کلاهبرداری یونیک

https://t.me/asnadeuniquefinanceponzi

**Comment by Webamoozcom** [comment]
![photo_2021-09-19_16-05-38](https://user-images.githubusercontent.com/87842536/133977219-c8375768-39eb-4367-af34-55dc0693950d.jpg)
این سران یونیک فاینس هم از آخور میخورن و هم از توبره

**Reply by Jabi-james** [reply]
> باید به هوش کسانی که در راس یونیک فایننس هستن آفرین گفت، یک کامیونیتی  با چند میلیون عضو، چرا برای ورود اعضا جدید و یا برداشت برای اعضای قدیمی از توکن اختصاصی خود استفاده نکنیم؟ خب توکن لانچ شد، قیمت مثلا ۱۰ سنت،،چگونه قیمت توکنی که خودمان درصد بالایی از آن را در اختیار داریم بالا ببریم؟چطور برای اعضا انگیزه ایجاد کنیم؟ پروموشن میگذاریم، سرمایه گذاران جدید حداقل با پرداخت ۳۵۰ دلار یک جایگاه هزار دلاری دریافت میکنن،یک آفر سه برابری،، ولی برای خرید باید از توکن استفاده کنید،، قانون عرضه و تقاضا، قیمت توکن بخاطر تقاضا زیاد میشود.از ۱۰ سنت به ۴ دلار،، ۴۰ برابر،،به بالا دستی ها هم ریوارد میدهیم، اما بصورت توکن.. حالا چه کنیم که از جیب خود برای پرداخت سودی که قول دادیم هزینه نکنیم؟ برای کسانی که میخواهند سود خود را دریافت کنند، توکن واریز میکنیم،توکنی که تنها هزینه اش برای ما هزینه لانچ آن بوده را بفروشید به ورودی های جدید  و پول خود را بردارید،پس اعضای قدیمی باید برای تبدیل توکن خود به تتر اعضای جدیدی را وارد کنند. حال اگر ورودی جدیدی نداشته باشیم و کسی نباشد که توکن را بخرد چه؟
> بسیار هوشمندانه است، نه؟

**Comment by saeedkousha** [comment]
![photo_2021-11-03_10-27-05](https://user-images.githubusercontent.com/91057696/140020361-d19e0fb5-00a1-403f-8da3-fbf369409700.jpg)
